.github/workflows/build-test-package.yml

```diff
@@ -13,6 +13,8 @@ on:
 jobs:
   cxx-build-workflow:
     uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-cxx.yml@v5.4.0
+    with:
+      itk-git-tag: dae9bef6fe16971fd8bd95766c0fc57591828e14
```

Comment:
@thewtex @dzenanz Do you know if this is the correct way to specify a specific revision of ITK? In this case, I picked https://github.com/InsightSoftwareConsortium/ITK/commit/dae9bef6fe16971fd8bd95766c0fc57591828e14 from the master branch.

Reply:
Yes, a git commit hash should work.

Reply:
Then maybe I did something wrong anyway 🤷 Just for the sake of the experiment, should I remove the "@v5.4.0" postfix when I use the `itk-git-tag`?

Reply:
@thewtex My Python environment does not recognize `itk.PointSetBase`, which was added last month to the master branch of ITK. Even though I did `pip install` a wheel from this pull request  🤷

Reply:
The Python wheels of base ITK are precompiled, and remote's wheels are built against that. `PointSetBase` has not made it into a [release](https://github.com/InsightSoftwareConsortium/ITKPythonPackage/tags) yet.

Reply:
Ah, thanks for explaining, @dzenanz 👍